bambi/interpret/logs.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from bambi.interpret import logger
+
+
+def log_interpret_defaults(func):
+    interpret_logger = logger.get_logger()
+    interpret_logger.setLevel(logging.INFO)
+
+    def wrapper(*args, **kwargs):
+        if not logger.messages:
+            return func(*args, **kwargs)
+
+        if func.__name__ == "set_default_values":
```

Comment:
I'm not sure if we should handle specific cases here. I think it would be better to allow the decorator to receive arguments and then determine what to do from those. 

Also, in this case, the function receives a dictionary and returns a dictionary. So you compare keys. Are there any other cases we may be interested?

Reply:
> I'm not sure if we should handle specific cases here. I think it would be better to allow the decorator to receive arguments and then determine what to do from those.

I agree. That would generalize better.

> Also, in this case, the function receives a dictionary and returns a dictionary. So you compare keys. Are there any other cases we may be interested?

There are two functions that compute default values. Both in `utils.py`.

1. [set_default_values](https://github.com/bambinos/bambi/blob/d001e069620a494bde730d69b47fb4c5ad3ba3e6/bambi/interpret/utils.py#L336) that computes default values for covariates specified in the model, but not passed to `conditional`.
2. [set_default_variable_values](https://github.com/bambinos/bambi/blob/d001e069620a494bde730d69b47fb4c5ad3ba3e6/bambi/interpret/utils.py#L89) that computes default `contrast` or `wrt` values for `comparisons` and `slopes`, respectively if the user does not pass a value (but they obviously need to pass the variable name of interest). 

These are the functions that I initially wanted to decorate. However, in other scenarios, e.g. a user passes a list of covariate names to `conditional`, but no values, then the following functions are called to compute default values:
1. [make_main_values](https://github.com/bambinos/bambi/blob/d001e069620a494bde730d69b47fb4c5ad3ba3e6/bambi/interpret/utils.py#L366) that computes main values based on original data using a grid of evenly spaced values for numeric predictors and unique levels for categoric predictors.
2. [make_group_values](https://github.com/bambinos/bambi/blob/d001e069620a494bde730d69b47fb4c5ad3ba3e6/bambi/interpret/utils.py#L378) that computes group values based on original data using unique levels for categoric predictors and quantiles for numeric predictors.

Knowing that a `np.linspace` or `np.quantile` would also be informative for the user. I will decorate these functions also since they compute default values as well.

Reply:
Sounds great